JSDOMParser.js

```diff
@@ -494,19 +494,14 @@
     get value() {
       return this._value;
     },
+    getValue: function() {
```

Comment:
Hm, there's already a `.value` getter, do we really need to define `getValue` as well?

Reply:
Probably not, I just figured it would be better to be consistent with `setValue`, although I suppose that could be a setter instead.